src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,167 @@
+import styled from "styled-components";
+// import { Button } from "../../reusableComponents/Button";
+import { useTranslation } from 'react-i18next'
+
+
+const FooterWrapper = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 10px;
+  max-width: 100%;
+  padding: 10px 50px 80px 50px;
+  align-items: center;
+  justify-content: center;
+
+  @media (min-width: 768px) {
+    grid-template-columns: 1fr 1fr 1fr;
+    padding: 10px 100px 80px 100px;
+    background-color: var(--footer-inactive-card-color);
+  }
+`;
+
+const FooterImage = styled.img`
+  width: 138px;
+  height: 104px;
+  margin: 0 auto;
+`;
+
+const ButtonWrapper = styled.div`
+ display: flex;
+ margin-left: 50px;
+@media (min-width: 768px) {
+  display: flex;
+ 
+}
+
+flex-direction: row;
+gap: 20px;
+justify-content: center;
+`;
+
+const TranslateButton = styled.button`
+  background: transparent;
+  border: none;
+  padding: 0;
+  cursor: pointer;
+`;
+
+const ContactInfo = styled.div`
+  display: none;
+
+  @media (min-width: 768px) {
+    display: block;
+  }
+`;
+const SocialMedia = styled.p`
+font-family: var(--heading-font-family);
+color:   #777b88;
+margin-top: 25px;
+`
+const SocialWrapper = styled.div`
+display: flex;
+flex-direction: row;
+
+`
+const Place = styled.div`
+display: flex;
+flex-direction: row;
+`
+const InfoParagraph = styled.p`
+font-family: Assistant;
+font-size: 18px;
+`
+const ContactFooter =styled.div`
+display: flex;
+flex-direction: row;
+justify-content: space-between;
+margin-top: 20px;
+margin-bottom: 20px;
+`
+// const MenuFooter = styled.div`
+// display: none;
+//   @media (min-width: 768px) {
+//     display: block;
+//   }
+// `
+
+// const MenuParagraph = styled.p`
+// font-family: var(--heading-font-family);
+// font-size: 18px;
+// `
+
+export const Footer = () => {
+
+  const { i18n } = useTranslation();
+ 
+
+  const changeLanguageFunc = (lng) => {
+    i18n.changeLanguage(lng);
+    console.log(`Changed language to: ${lng}`);
+
+}
+const englishFunc = () => {
+  changeLanguageFunc("en");
+  console.log("Clicked on English button");
+}
+
+  const swedishFunc = () => {
+  console.log("Clicked on Swedish button");
+  changeLanguageFunc("se");
+  
+}
```

Comment:
This looks easy - so I'm guessing you have worked a lot to get it to work 😅 Or maybe this is the only easy part of the translations :P